components.css

```diff
@@ -18,6 +18,7 @@
     --primarycolor:#35589A;
     --ButtonLink:#99FEFF;
     --background-card:white;
+    --textgray:gray;
```

Comment:
Suggestion: Since you're using hex before for declaring color variables, you may use it here as well for consistency.

Reply:
will take care from next time